test/spec/features/ContextPadTracking.spec.js

```diff
@@ -4,6 +4,10 @@ import {
   getBpmnJS
 } from 'test/TestHelper';
 
+import {
+  compareEvents
```

Comment:
Two questions: Can we find a better name, for example `expectEvent(event, expectedEvent)`? Is it necessary to do JSON stringification to compare? I'd rather use the existing deep equality helpers (cf. https://github.com/bpmn-io/bpmn-js-tracking/pull/36/files#diff-732dfd6a397038898d411e5ccf24c6c72c25e60fe66709c7c602b99290831d2aL146) if possible; these give us a proper diff.

Reply:
I just found we have some examples using a [`jsonEqual` assertion ](https://github.com/bpmn-io/bpmn-js-element-templates/blob/main/test/TestHelper.js#L42) but it also uses JSON stringification. Do you mean something like this or something else entirely?

Reply:
My user perspective is: Let's write readable tests.

That means that I know if a utility is asserting something. Of course a custom assertion is great, however sometimes a simple `expectXYZ` works, too.

What does not work well (for me) is `compareXYZ` because that (1) does no blow up and (2) [returns an integer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#parameters) (-1 for smaller, 0 for equal and 1 for bigger)

Reply:
I understand. I think with my latest change (based on my the example i showed) it's clear what we are asserting